storybook/stories/WalletAlert/index.js

```diff
@@ -19,18 +19,21 @@ StoryWalletAlert.defaultProps = {
   children: null,
   actions: [
     {
+      id: 1,
       isMain: true,
       title: 'Main test title',
       onPress: () => { },
     },
     {
+      id: 2,
       isMain: false,
       title: 'First test title',
       onPress: () => { },
     },
     {
+      id: 3,
       isMain: false,
-      title: 'Second test title',
+      title: 'ECgludvYJsDsyCMvZC3Vx0YEtu1mtS3RXKgBVHXNyUA9vwxaT5SKvJuICG1x2DpSM7HxHVa8CGTLHXWGIVMPRuVJsN0mTPXJP0qowaLdKkLSCnKTMbRhPv7KgeW8jOM5n8fOIrzCTJ9UNWx3RM4uJt35WYMe2IW2Qw5DKnwCTenbt5Byc1LbLYOX9ubb9azS8R79NwX5ZUu3EIEIjqoXmlwIKRZqb0Ug0KtHfqPeUA6xeAwK1quPGjGtqqjuwtH',
```

Comment:
It is better to add something like this: "This is very long title for testing purposes ECgludvYJsDsyCMvZC3Vx0YEtu1mtS3RXKgBVHXNyUA9vwxaT5SKvJuICG1x2DpSM7HxHVa8CGTLHXWGIVMPRuVJsN0mTPXJP0qowaLdKkLSCnKTMbRhPv7KgeW8jOM5n8fOIrzCTJ9UNWx3RM4uJt35WYMe2IW2Qw5DKnwCTenbt5Byc1LbLYOX9ubb9azS8R79NwX5ZUu3EIEIjqoXmlwIKRZqb0Ug0KtHfqPeUA6xeAwK1quPGjGtqqjuwtH"

Because behaviour is different if title has spaces or hasn't.